virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -85,6 +85,43 @@ endif::openshift-rosa,openshift-dedicated[]
 
 . xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#attaching-vm-to-ovn-secondary-nw[Connect the VM to the OVN-Kubernetes secondary network] by adding the network details to the VM specification.
 
+// Hiding from ROSA/OSD as Linux Bridge is not supported
+ifndef::openshift-rosa,openshift-dedicated[]
+[id="comparing-bridge-localnet"]
+=== Comparing Linux bridge CNI and OVN-Kubernetes localnet topology
+
+The following table provides a comparison of features available when using the Linux bridge CNI versus the localnet topology for an OVN-Kubernetes plug-in:
```

Comment:
```suggestion
The following table provides a comparison of features available when using the Linux bridge CNI versus the localnet topology for an OVN-Kubernetes plugin:
```
Consistent with named term in core docs: https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html